TESTS.md

```diff
@@ -292,6 +292,28 @@ test
   ,)
 ```
 
+# Record syntax
+
+Pattern matching, short
+
+```haskell
+fun Rec {alpha = beta, gamma = delta, epsilon = zeta, eta = theta, iota = kappa} = do
+  beta + delta + zeta + theta + kappa
```

Comment:
Sorry to nitpick, but why is there no space *after* the `{` either (and before the `}`)? This is inconsistent with multi-line formatting.